index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/styles.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header class="container">
+       <a class="name">John Doe</a>
```

Comment:
`a` tags are anchors which need to link the user somewhere. 

- [ ] Kindly add for the moment to this `a` tag an href attribute with the value of '#'.  It should be something like this: `<a href='#'>...</a>`;